manifests/configmap.yaml

```diff
@@ -7,6 +7,7 @@ data:
   cluster_labels: application:spilo
   cluster_name_label: version
   pod_role_label: spilo-role
+  toleration: "key:postgres,operator:Exists,effect:NoSchedule"
```

Comment:
This should  not be in the default configmap as this one is also used for minikube example and should work in most clusters which we for now assume do not have the postgres tain/toleration in place.

Reply:
Thanks for your reply. I thought about that and I included it for documentation purposes but understand your concern.
It does work with the toleration though as it does not require the taint on the node to get scheduled. But it's not really clean.
Any proposal where I can document it?

Reply:
I thought using "PreferNoSchedule" is the soft border, but anyways, feel free maybe to put this into the readme as a small sub section after testing, as I believe this a valid use case/question others may have too.